package.json

```diff
@@ -20,8 +20,7 @@
     "npm:tag:beta": "npm run npm:tag -- beta",
     "npm:tag:latest": "npm run npm:tag -- latest",
     "notify:docs": "node ./scripts/notify-docs/published-ui-kit.js",
-    "doc:generate": "lerna run doc:generate",
-    "lockLernaDependencies": "lerna exec \"npm install --package-lock-only\""
```

Comment:
We have to rethink running this command on `setup` because it fails if a package is not yet published.

Context: https://github.com/coveo/ui-kit/pull/1013#discussion_r678541659

Reply:
We should check first if the `package-lock.json` files are fixed in the current or the latest version of lerna. If so, we no longer need `lockLernaDependencies`. If not, we must keep it to avoid pushing bad `package-lock.json` files or we must make the CI replace package-lock.json files.